docs/networking/split-horizon_scenarios.md

```diff
@@ -0,0 +1,95 @@
+# Split-Horizon DNS for Percona Operator for MongoDB scenarios
+
+
+## Create Split-Horizon DNS config
+
+A Percona Everest administrator can create a set of base domains and generate the necessary TLS certificates in advance. This allows all Percona Everest users to use these domains later without the need to define custom domain names each time they create a MongoDB cluster.
+
+Percona Everest provides an interface to create a Split-Horizon DNS configuration that defines the necessary relationships and stores them for future use.
+
+The following applies to Split-Horizon DNS configs:
+
+- Each config has a unique name within the selected namespace.
+
+- It can only be applied to the Percona Operator for MongoDB Engine. The other database engines do not support this feature.
+
+- Each config is associated with a specific namespace, which the user defines during creation.
+
+**Split-Horizon DNS config parameters**
+
+| Configuration Parameter      | Description                                                                                                         |
+|-------------------------------|---------------------------------------------------------------------------------------------------------------------|
+| **Name**                      | Each **Split-Horizon DNS config** has a unique name within the selected namespace.                                   |
+| **Namespace**                 | Kubernetes namespace to which this config is applicable. Defined by the user during config creation.                 |
+| **Base domain name**          | Domain name suffix used for generating full domain names for each Pod in the Replica Set. This field is also used as the **Split-Horizon DNS config** name. |
+| **TLS certificates**          | TLS certificate files for the provided base domain. See [instructions](https://docs.percona.com/percona-operator-for-mongodb/tls-manual.html#__tabbed_1_1). <br>- `cert.pem` - certificate<br>- `cert-key.pem` - certificate key<br>- `ca.pem` - certificate authority file |
```

Comment:
please check carefully with the spec - there are no `cert.pem` and `cert-key.pem` used anymore

Reply:
Fixed